scripts/govcms-deploy

```diff
@@ -53,7 +53,7 @@ common_deploy () {
   if [[ "$LAGOON_ENVIRONMENT_TYPE" = "development" && "$GOVCMS_DEPLOY_WORKFLOW_CONTENT" = "import" ]]; then
     echo "Performing content import."
     # shellcheck disable=SC2086
-    drush --alias-path=/etc/drush/sites sql:sync @ci.prod @self -y
+    drush --alias-path=/app/drush/sites sql:sync @govcms.prod @self -y
```

Comment:
govcms8lagoon, govcms8-scaffold-paas and the deploy/test scripts in this project all refer to the alias as `govcms.prod` - updating here for consistency